geoh5py/io/h5_writer.py

```diff
@@ -322,17 +318,19 @@ def update_field(
 
             if entity_handle is None:
                 return
-
-            if attribute in [
-                "concatenated_attributes",
-                "metadata",
-                "options",
+            if attribute in ["concatenated_attributes", "options"]:
```

Comment:
```suggestion
            if attribute in ["concatenated_attributes"]:
```

Reply:
No, "options" is pointing to group or concatenator object.

Reply:
you're right, options and metadata are the same for the tests